sdk/storage/Azure.Storage.Common/tests/AesGcmTests/Azure.Storage.Common.Tests.AesGcm.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
```

Comment:
I meant to only put net461, since the other target frameworks are not meant to be added. But we're not allowed to compile without the other ones. If someone can point me to only targetting certain frameworks let me know.

Reply:
It might be good to run tests on runtimes that don't support this feature and simply verify that we throw the right exceptions.